### PR TITLE
Home page - (The Next Web Clone) - Milestones 1-6

### PR DESCRIPTION
**Pull request by SamArsanios**
In this project:

- We created mobile, tablet, and pc responsive designs for the site using media queries.
- created sections and used flexbox, inlines to align pages.

### CODE REVIEW

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳 ♲
Hi @udberg and @SamArsanios, I am [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  👏👏  I appreciate your hard work in replicating thenextweb.com. Most requirements have been done 👌,  kindly fix the following issues.

[REQUIRED]

- [x] On medium screens, the navbar should contain the thenextweb.com logo at the left of the header.

- [x] The TNW logo on the top news should disappear on medium and small screens.

- [x] The hamburger menu should appear on the right top corner of small screens.

- [x] Fix the spacing on the top news left side

- [x] Remove DS_Store file from the GitHub repo and Add it inside .gitignore

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_modules/
+.hintrc
+.stylelintrc.json
```
Comment (descholar-ceo) [inline .gitignore]: Remove this `.hintrc` and `.stylelintrc.json` files from gitignore :point_right: They are linting files, so you need them in your repository

File: index.html [hunk 2]
```diff
@@ -0,0 +1,502 @@
...
+                <svg height="20" width="20" viewBox="0 0 20 20">
+                    <path d="M7,13,1,19Zm0-9.95A7,7,0,1,1,5,8,7,7,0,0,1,7.05,3.05Z" fill="none" stroke="currentColor"
+                        stroke-linecap="round" stroke-linejoin="round" stroke-width="2" />
+                </svg>
+                <a href="#">
+                    <i class="fa fa-bars" style="width: 40px; background-color: #333;"></i>
+                </a>
+            </div>
+        </div>
+    </section> -->
```
Comment (descholar-ceo) [inline index.html]: Remove the commented codes

File: index.html [hunk 3]
```diff
@@ -0,0 +1,502 @@
...
+                    <div><span>Monday — November 30, 2020</span></div>
+
+                    <span>WORLD</span>
+                    <a
+                        href="https://thenextweb.com/syndication/2020/11/30/how-to-check-your-christmas-trees-carbon-footprint/">
+                        <h2>How
+                            to check your Christmas tree’s carbon footprint</h2>
+                    </a>
+                    <p>by The Conversation 39 minutes ago 0</p>
+                </div> -->
```
Comment (descholar-ceo) [inline index.html]: This commented code also should be removed

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @SamArsanios and @udberg  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]
- [x] The top three articles should take the whole width of the screen each one, learn the screenshots below to see how the original page differ from your clone on mobile screen and work on it accordingly

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-03 13-59-38| Screenshot from 2020-12-03 14-00-23|

:point_right: Could you see the difference :question: :question: :question: 
- [x]  The TNW logo on the top news should disappear on medium and small screens.
- [x] Implement all changes I suggested in inline comments in your codes

# [Optional changes]
- [x] Your commit messages should be in imperative mode or present tense, _i.e_ you should have written `Fix linter errors` instead of writting `Fixed linter errors`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

___
_If anything seems to be unclear to you, comment on this Pull Request and mention me as [@descholar-ceo](https://github.com/descholar-ceo) I will receive a notification and will respond._

| [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Review by layzrd — APPROVED** [review]
### Status: Approved 🙌 🎇
Hi @udberg and @SamArsanios, I am [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR. Congratulations 🎊 👏, you did very well on this project. 👏👏  I appreciate your hard work in replicating thenextweb.com.

Now, you can move ahead.

Please don't forget to merge to master. Good luck with the next projects!

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Comment by SamArsanios** [comment]
> ### Status: Approved 🙌 🎇
> Hi @udberg and @SamArsanios, I am [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR. Congratulations 🎊 👏, you did very well on this project. 👏👏 I appreciate your hard work in replicating thenextweb.com.
> 
> Now, you can move ahead.
> 
> Please don't forget to merge to master. Good luck with the next projects!
> 
> Happy coding 👨‍💻
> [Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

File: index.html (hunk 3 above)
Comment (SamArsanios) [reply index.html]: Thanks a lot for the review

**Pull request merged**